src/content/structured/components/checkboxes/code.mdx

```diff
@@ -541,10 +532,7 @@ export const snippetsIndeterminate = [
   },
 ];
 
-<ComponentPreview
-  snippets={snippetsIndeterminate}
-  style={{ marginTop: "1.5rem" }}
->
+<ComponentPreview snippets={snippetsIndeterminate}>
```

Comment:
The component preview buttons have turned blue, from their original blavk

Reply:
Monochrome prop is added in the buttons, but doesn't seem to be in the package yet so will display as blue for a bit. Should hopefully be fixed after the next release